src/NibLogo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {LOGO_THEME_COLORS} from './constants';
+
+export default function NibLogo(props) {
+  return (
+    <svg
+      x="0px"
+      y="0px"
+      width="42px"
+      height="22px"
+      viewBox="2.5 0.2 42 22"
+      aria-labelledby="title"
+      aria-describedby="desc"
+      role="img"
+    >
+      <title id="title">nib</title>
```

Comment:
these aren't necessary for svg are they? might be worth running it through a svg minifier.